docs/running-without-go.md

```diff
@@ -3,15 +3,18 @@
 `gotestsum` may be run without Go as long as the package to be tested has
 already been compiled using `go test -c`, and the `test2json` tool is available.
 
-The `test2json` tool can be compiled from the Go source tree:
+The `test2json` tool can be compiled without installing go:
 
 ```sh
-export GOVERSION=1.13.5
-curl -Lo go.zip "https://github.com/golang/go/archive/go${GOVERSION}.zip"
-unzip go.zip
-rm -f go.zip
-cd go-go${GOVERSION}/src/cmd/test2json/
-env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags="-s -w" .
+GOVERSION=1.13.5
```

Comment:
```suggestion
GOVERSION=1.17.6
```

We may as well update this to something recent while we are in here making changes.